packages/web/src/client/components/Blocks/BlockTestimonial.tsx

```diff
@@ -41,6 +41,9 @@ const TestimonialWrapper = styled(FadeIn, {
   '@tabletUp': {
     maxWidth: '$centeredParagraph',
   },
+  '@desktopUp': {
+    maxWidth: '800px',
```

Comment:
Is there a reason that a. You didn't use the theme token & b. Only applied this on Desktop, not changing the tablet setting below?

Reply:
@joshuaellis on tablet it looked so wide with barely any space left on the sides. Also, Myles mentioned specifically the desktop in the ticket, so these both combined made me thing it might be a better idea to apply it there.
The `centeredParagraph` has one specific value 65.3rem and we want 800px specifically.

Reply:
ah, didn't realise the `documentWrapper` was the same size, cool :)